src/wp-includes/class-wp-theme-json.php

```diff
@@ -1705,12 +2040,12 @@ function( $pseudo_selector ) use ( $selector ) {
 		 * Otherwise just compute the styles for the default selector as normal.
 		 */
 		if ( $pseudo_selector && isset( $node[ $pseudo_selector ] ) &&
-			array_key_exists( $current_element, static::VALID_ELEMENT_PSEUDO_SELECTORS )
-			&& in_array( $pseudo_selector, static::VALID_ELEMENT_PSEUDO_SELECTORS[ $current_element ], true )
+				array_key_exists( $current_element, static::VALID_ELEMENT_PSEUDO_SELECTORS ) &&
+				in_array( $pseudo_selector, static::VALID_ELEMENT_PSEUDO_SELECTORS[ $current_element ], true )
```

Comment:
Should this have changed? Looks like some formatting weirdness slipped in.

Reply:
Oh, thanks! I think that's just a whitespace issue from my rebase. I'll fix this up.